bundles/org.openhab.binding.salus/README.md

```diff
@@ -75,11 +75,12 @@ removed.
-|-----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| it600-temp-channel          | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
+| Channel                    | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
+|----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| it600-temp-channel         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
 | it600-expected-temp-channel | Number:Temperature | RW         | Sets the desired temperature in the room                                                                                                                                                                                                                                                        |
-| it600-work-type-channel     | String             | RW         | Sets the work type for the device. OFF - device is turned off MANUAL - schedules are turned off, following a manual temperature set, AUTOMATIC - schedules are turned on, following schedule, TEMPORARY_MANUAL - schedules are turned on, following manual temperature until the next schedule. |
+| it600-work-type-channel    | String             | RW         | Sets the work type for the device. OFF - device is turned off MANUAL - schedules are turned off, following a manual temperature set, AUTOMATIC - schedules are turned on, following schedule, TEMPORARY_MANUAL - schedules are turned on, following manual temperature until the next schedule. |
+| it600-running-state     | Switch             | RO         | Is the device running |
```

Comment:
```suggestion
| running-state     | Switch             | RO         | Is the device running |
```
According to the it600.xml this channel id is different. Looks like you added the type id, while it should be the channel id here.